main.tf

```diff
@@ -217,8 +217,9 @@ resource "aws_autoscaling_group" "agent_asg" {
         version            = var.agent_lt_version
       }
 
+      # Why???
```

Comment:
Replace the question with this comment: 'The mixed instance policy allows multiple override instance types'